unitycatalog-ai/integrations/llama_index/pyproject.toml

```diff
@@ -28,11 +28,13 @@ build-backend = "hatchling.build"
 
 [tool.hatch.build]
 include = [
-  "src/ucai_llamaindex/*"
+  "src/unitycatalog/ai/llama_index/*"
 ]
 
 [tool.hatch.build.targets.wheel]
-packages = ["src/ucai_llamaindex"]
+sources = ["src"]
+only-include = ["src/unitycatalog/ai/llama_index"]
+packages = ["src/unitycatalog"]
```

Comment:
Seems like this is not needed https://hatch.pypa.io/1.12/config/build/#packages

Reply:
Good catch! Removed redundant option :)